meteor/client/ui/RundownView.tsx

```diff
@@ -106,7 +106,7 @@ import { RundownLayoutsAPI } from '../../lib/api/rundownLayouts'
 import { TriggersHandler } from '../lib/triggers/TriggersHandler'
 import { SorensenContext } from '../lib/SorensenContext'
 import { PlaylistTiming } from '@sofie-automation/corelib/dist/playout/rundownTiming'
-import { CASPARCG_RESTART_TIME } from '@sofie-automation/shared-lib'
+import { CASPARCG_RESTART_TIME } from '@sofie-automation/shared-lib/dist/core/constants'
```

Comment:
What is the reason for importing from the `dist` folder?

Reply:
@Julusian can probably answer this even better, but as I understood it there is a bug in Typescript that prevents us from importing from the proper path (ie `@sofie-automation/shared-lib/core/constants`).

It would of course be possible to add exports to the main index.ts-file in shared-lib and just import everything from `@sofie-automation/shared-lib`, but that might become a hassle later as the library grows and contains many different kinds of types/constants/helper functions.

So we opted to import using the `/dist/` path for now, and when the Typescript-bug is solved we can remove it later.

Reply:
Yes, I believe we need typescript 4.7 (we currently use 4.5) to be able to drop the `/dist/` portion of the import path. https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/#package-json-exports-imports-and-self-referencing
We felt it was better to have this there temporarily (we made the same decision when doing the job-worker, back when support for this was aiming for 4.5), rather than to export everything through the index.ts, as that would require us to rename a lot of the exports to ensure their name/purpose is clear without the context of file you are importing from

Reply:
Isn't the bug just that Typescripts suggest the wrong path for auto-import? The right path should still work?

Nevertheless. Glad is temporary

Reply:
When compiling it also fails to resolve the correct path, causing it to report every import as missing a file.
The right path does work at runtime, but with typescript unable to resolve those paths at all, we have been a little stuck